lib/clipboard.zsh

```diff
@@ -0,0 +1,70 @@
+# System clipboard integration
+#
+# This file has support for doing system clipboard copy and paste operations
+# from the command line in a generic cross-platform fashion.
+#
+# On OS X and Windows, the main system clipboard or "pasteboard" is used. On other
+# Unix-like OSes, this considers the X Windows CLIPBOARD selection to be the
+# "system clipboard", and the X Windows `xclip` command must be installed.
+#
+# When using X Windows, copied data will only be good for a single paste
```

Comment:
Oops! This is just a mistake I made reading the `man xclip` documentation. I thought it defaulted to `-loops 1` and misread this section

> The default action is to silently wait in the background for X selection requests (pastes) until another X application places data in the clipboard, at which point xclip exits silently.

as saying it waits until another application *reads from* the clipboard.

I'll remove it.